pkg/kubelet/cm/devicemanager/checkpoint/checkpointv1.go

```diff
@@ -63,9 +63,9 @@ type DataV1 struct {
 }
 
 // NewV1 returns an instance of Checkpoint, in V1 (k8s <= 1.19) format.
-// Users should avoid creating checkpoints in formats different than the most recent one,
+// Users should avoid creating checkpoints in formats different from the most recent one,
 // use the old formats only to validate existing checkpoint and convert them to most recent
-// format. The only exception should be test code.
+// format. The only exception should be tested code.
```

Comment:
I think `test code` is correct here as `tested` code implies the code that is being tested, not the code that is doing the testing `*_test.go`

Reply:
I've reverted this change.